src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+interface Props {
```

Comment:
[Not critical]
It's better to name this type something like `TodoListProps`, so that the name describes this type the most precisely.